### ISSUE TITLE
Missing trailing '/' in generated Alias when pointing mod_wsgi-express's --url-alias option to a directory

### ISSUE THREAD

**Issue by sevens** [GrahamDumpleton/mod_wsgi#104]
When using `mod_wsgi-express start-server` with the `--url-alias` option pointing to a directory, the generated `Alias` config line seems to be missing a trailing `/` in its `directory-path` part.  This results in the contents of the aliased folder not being accessible, as the local filesystem path is also constructed without the trailing `/`.

E.g.:
```bash
$ mod_wsgi-express start-server --url-alias foo/bar/ /home/user/src/bar/ $some_other_options
$ curl http://localhost:8000/foo/bar/baz
```
results in the following output:
```
[access_compat:error] [pid ...:tid ...] [client ::1:54986] AH01797: client denied by server configuration:
/home/user/src/barbaz
```

Modifying the generated `httpd.conf` file and changing the `Alias` line from
```apache
Alias 'foo/bar/' '/home/user/src/bar'
```
to
```apache
Alias 'foo/bar/' '/home/user/src/bar/'
```
and manually starting apache (using the auto-generated `apachectl` script) fixes the issue.

There doesn't seem to be any difference in supplying the directory to `--url-alias` with or without the trailing slash nor quoting it (with trailing slash).

Using mod_wsgi 4.4.21, Python 2.7.10 (64 bits), Apache HTTP 2.4.16 on Linux (4.1.6, 64 bits).

**Comment by GrahamDumpleton** [comment]
Yep. I'll need to handle this in a few places probably. The issue is that I use ``os.path.normpath()`` on the target directory but unfortunately that strips the trailing slash. This causes a problem because the trailing slash is significant to Apache in this case.

You could in then interim use:

```
mod_wsgi-express start-server --url-alias /foo/bar /home/user/src/bar
```

That is, don't use trailing slash on either.

By not having a trailing slash on the URL, it does mean a request to '/foo/bar' will auto redirect to '/foo/bar/' which can be useful if you have directory index files or browsing enabled. Perhaps not relevant to you though.

The only issue with using '/foo/bar' is that it is a prefix match and so risk it can interfere with other URLs with common prefix.

**Comment by sevens** [comment]
Hi Graham,

Thanks, that works (luckily the prefix matching isn't an issue in my case).  Isn't it possible though to use the same logic that seems to detect that a `<Directory>` section is needed for the path given to `--url-alias` to determine if a trailing `/` is needed?

**Comment by GrahamDumpleton** [comment]
Not sure what you mean, but all that is needed is instead of the code saying:

```
                target = os.path.abspath(target)
```

it needs to do something like:

```
                original_target = target
                target = os.path.abspath(target)

                if original_target[-1] == '/':
                    target = target + '/'
```

It has been a rather busy week so been hard to get around to spend some time on it.

**Comment by sevens** [comment]
Ok, no problem, current work-around works fine for now.  I guessed the config was generated per section depending on various options the user supplied, thus generating a `<Directory>` block only if needed, but I checked the code yesterday (for some other issues) and saw it was a pre-defined string.

**Comment by GrahamDumpleton** [comment]
Fixed for 4.4.22.
